.github/workflows/trivy-scan.yml

```diff
@@ -0,0 +1,88 @@
+name: Trivy CVE Dependency Scanner
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  scan-all:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v1
+        with:
+          go-version: "1.17"
+      - name: Build the kapp-controller artifacts
+        run: |
+          ./hack/install-deps.sh
+          ./hack/build.sh
+
+          # docker image
+          docker build -t docker.io/carvel/kapp-controller:${{ github.sha }} .
```

Comment:
hmm... it seems that we are building this image from scratch instead of analyzing last release image, which i think would mean that we would always be getting latest updates to images -- so no cves caught.

Reply:
The outcome from our conversation RE this comment:

We will create another daily scan that gets the latest released docker image and scans it.

Reply:
This ended up being more complicated, getting the scan results into the security tag based on a tag doesn't seem to work for me. So I am going to move this out into another issue. I think having a daily run against HEAD is good enough that I don't want to block on previously released scanning.